mlir/lib/Target/LLVMIR/ModuleImport.cpp

```diff
@@ -952,6 +962,42 @@ ModuleImport::getOrCreateNamelessSymbolName(llvm::GlobalVariable *globalVar) {
   return symbolRef;
 }
 
+LogicalResult ModuleImport::convertAlias(llvm::GlobalAlias *alias) {
+  // Insert the global after the last one or at the start of the module.
+  OpBuilder::InsertionGuard guard(builder);
+  if (!aliasInsertionOp)
+    builder.setInsertionPointToStart(mlirModule.getBody());
+  else
+    builder.setInsertionPointAfter(aliasInsertionOp);
+
+  Type type = convertType(alias->getValueType());
+  AliasOp aliasOp = builder.create<AliasOp>(
+      mlirModule.getLoc(), type, convertLinkageFromLLVM(alias->getLinkage()),
+      alias->getName(),
+      /*addr_space=*/alias->getAddressSpace(),
```

Comment:
Can you check if this just falls back to reading the address space from the type or if this is really a value? As stated before, the LangRef does not mention any address space for aliases (not that this means anything...).

Reply:
Gets it from the type:
```
  unsigned getAddressSpace() const {
    return getType()->getAddressSpace();
  }
```